cpu/cc26x0/periph/rfc.c

```diff
@@ -114,8 +114,9 @@ void rfc_prepare(void)
     PRCM->CLKLOADCTL = CLKLOADCTL_LOAD;
     while (!(PRCM->CLKLOADCTL & CLKLOADCTL_LOADDONE)) ;
 
-    PRCM->PDCTL0RFC = 1;
-    PRCM->PDCTL1RFC = 1;
+    /* RFC POWER DOMAIN */
+    PRCM->PDCTL0 |= PDCTL0_RFC_ON;
+    PRCM->PDCTL1 |= PDCTL1_RFC_ON;
```

Comment:
I've decided to use PDCTL0 and PDCTL1 rather than their alias registers. This way we don't need to have additional declaration. I don't know why it worked before though :octopus: